unit-tests/run-unit-tests.py

```diff
@@ -40,6 +40,18 @@ def usage():
     print( '                         with what tags it has' )
     print( '        --no-exceptions  do not load the LibCI/exceptions.specs file' )
     print( '        --context        The context to use for test configuration' )
+    print()
+    print( 'Examples:' )
+    print( 'Running: python <path-to-exe-dir> run-unit-tests.py -r cpp -t log' )
+    print( "    Will run all tests whose name contain 'cpp' and who have the tag 'log' while searching for the " )
+    print( "    exe files in the provided directory. Each test will create it's own .log file to which it's " )
+    print( "    output will be written. The output to stdout will have the names of all the tests that ran." )
+    print( 'Running: python run-unit-tests.py --list-tests --list-tags' )
+    print( "    Will find all test source files and print for each one what tags it has in the following format " )
+    print( '    <test-name> has tags: <tags-separated-by-spaces>' )
+    print( 'Running: python run-unit-tests.py -s' )
+    print( '    Will try to deduce the directory holding the exe files by searching for test executables in ' )

```

Comment:
The whole thing about .exe files versus source files is too much for examples... let's just provide a bare-bones example that says "Runs all tests, but direct their output to the console rather than log files"

Reply:
Not sure I understand. You want to just leave out the part of the script finding the build directory on it's own?
If we do so the user won't understand why in the later example we provide a directory in the command line...

Reply:
Because it's optional. Usually nobody even knows it's an option -- I never run with it. For a basic example, we shouldn't overcomplicate.

Reply:
OK, so just change the explanation, right?
I'm also pleased to hear that a change I made is so popular that no one knows about the other option :)